### PR TITLE
Upgrade to @react-native-async-storage/async-storage

### PR DESCRIPTION
**Pull request by jamesaucode**
*Issue #, if available:*
#654
#642 

*Description of changes:*
Upgrade from `@react-native-community/async-storage` to `@react-native-async-storage/async-storage`

Also upgrading `@redux-offline/redux-offline` to `2.6.0-native.1` as the old version depends on `@react-native-community/async-storage`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/aws-appsync/package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "uuid": "3.x"
   },
   "peerDependencies": {
-    "@react-native-community/async-storage": "^1.11.0",
+    "@react-native-async-storage/async-storage": "^1.11.0",
     "@react-native-community/netinfo": "^5.0.0"
   }
```
Comment (iartemiev) [inline packages/aws-appsync/package.json:43]: Let's remove these `peerDependencies` altogether. See https://github.com/awslabs/aws-mobile-appsync-sdk-js/pull/675 for additional info.

**Review by iartemiev — CHANGES_REQUESTED** [review]

**Review by iartemiev — APPROVED** [review]
LGTM 👍🏻

**Comment by andidev** [comment]
This is really nice to hear its merged! When can we expect a release?

**Comment by iartemiev** [comment]
Hey @andidev, we just released the change this morning in `aws-appsync@4.1.2`

**Pull request merged**